### PR TITLE
Design tidy up

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: assets/scss/buttons-links.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    top: auto;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+    z-index: -999;
+  }
+
+  &.skip-main:focus, &.skip-main:active {
+    color: #fff;
+    background-color: #000;
```
Comment (monkemedia) [inline assets/scss/buttons-links.scss]: Is there a variable for black?

Comment (monkemedia) [reply assets/scss/buttons-links.scss]: and white?

File: assets/scss/buttons-links.scss [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    left: auto;
+    top: auto;
+    width: 30%;
+    height: auto;
+    overflow: auto;
+    margin: 10px 35%;
+    padding: 5px;
+    border-radius: 15px;
+    border: 4px solid yellow;
```
Comment (monkemedia) [inline assets/scss/buttons-links.scss]: and yellow?

File: assets/scss/buttons-links.scss (hunk 1 above)
Comment (mirjamverloop) [reply assets/scss/buttons-links.scss]: Ah, yes, that is the skip to main accessibility link. It now uses variables.

**Pull request merged**